src/container-manager/docker-files/lumino-node/assets/lumino-docker-fix.py

```diff
@@ -0,0 +1,282 @@
+    Returns:
+        func(): result of func, if truth value, or None"""
+    res = func()
+
+    if res:
+        return res
+
+    if wait_for:
+        deadline = time.time() + wait_for
+        while not res and time.time() <= deadline:
+            gevent.sleep(sleep_for)
+            res = func()
+
+    else:
+        while not res:
+            gevent.sleep(sleep_for)
+            res = func()
+
+    return res
+
+
+def is_frozen() -> bool:
+    return getattr(sys, "frozen", False)
+
+
+def split_in_pairs(arg: Iterable) -> Iterable[Tuple]:
+    """ Split given iterable in pairs [a, b, c, d, e] -> [(a, b), (c, d), (e, None)]"""
+    # We are using zip_longest with one clever hack:
+    # https://docs.python.org/3/library/itertools.html#itertools.zip_longest
+    # We create an iterator out of the list and then pass the same iterator to
+    # the function two times. Thus the function consumes a different element
+    # from the iterator each time and produces the desired result.
+    iterator = iter(arg)
+    return zip_longest(iterator, iterator)
+
+
+def create_default_identifier() -> int:
+    """ Generates a random identifier. """
+    return random.randint(0, constants.UINT64_MAX)
+
+
+def merge_dict(to_update: dict, other_dict: dict) -> None:
+    """ merges b into a """
+    for key, value in other_dict.items():
+        has_map = isinstance(value, collections.Mapping) and isinstance(
+            to_update.get(key, None), collections.Mapping
+        )
+
+        if has_map:
+            merge_dict(to_update[key], value)
+        else:
+            to_update[key] = value
+
+
+def optional_address_to_string(
+    address: Optional[Union[Address, TokenAddress]] = None,
+) -> Optional[str]:
+    if address is None:
+        return None
+
+    return to_checksum_address(address)
+
+
+def safe_gas_limit(*estimates: int) -> int:
+    """ Calculates a safe gas limit for a number of gas estimates
+    including a security margin
+    """
+    assert None not in estimates, "if estimateGas returned None it should not reach here"
+    calculated_limit = max(estimates)
+    return int(calculated_limit * constants.GAS_FACTOR)
+
+
+def to_rdn(rei: int) -> float:
+    """ Convert REI value to RDN. """
+    return rei / 10 ** 18
+
+
+def block_specification_to_number(block: BlockSpecification, web3: Web3) -> BlockNumber:
+    """ Converts a block specification to an actual block number """
+    if isinstance(block, str):
+        msg = f"string block specification can't contain {block}"
+        assert block in ("latest", "pending"), msg
+        number = web3.eth.getBlock(block)["number"]
+    elif isinstance(block, T_BlockHash):
+        number = web3.eth.getBlock(block)["number"]
+    elif isinstance(block, T_BlockNumber):
+        number = block
+    else:
+        if __debug__:
+            raise AssertionError(f"Unknown type {type(block)} given for block specification")
+
+    return BlockNumber(number)
```

Comment:
Actually this file didn't change in a lot of time and is a raiden code more than a lumino code, i hear you but i didn't want to make a change on the lumino source code since we are releasing, making a script or using the same as we have right now implies that we need to parse the output and a lot of interactions that adds more complexity to the process than just copy and paste a file.